SSA/Projects/RISCV64/ParseAndTransform.lean

```diff
@@ -35,11 +35,11 @@ def parseAsRiscv (fileName : String ) : IO UInt32 := do
     match icom? with
     | none => return 1
     | some (Sigma.mk _Γ ⟨_eff, ⟨_retTy, c⟩⟩) => do
-      IO.println s!"{toString c}"
+      IO.println s!"{Com.toPrint c}"
       return 0
+
 private def test_simple := [RV64_com| {
   ^bb0(%e1 : !i64, %e2 : !i64 ):
   %1 = add %e1, %e2 : !i64
        ret %1 : !i64
 }]
-#eval! test_simple
```

Comment:
This change was unrelated. I upstreamed it separately in https://github.com/opencompl/lean-mlir/pull/1555.